### PR TITLE
Prod 62:Create application, delete comments, add PR Layout, configure .gitignore and rubocop.yml

### PR DESCRIPTION
**Pull request by VitaliiChverenko**
screenshot from 2018-04-02 17-00-19

### CODE REVIEW

**Comment by shevchukm** [comment]
:+1:

**Comment by imysheniuk** [comment]
:+1:

**Comment by okohuch** [comment]
I do not understand what this PR is for. Could you give it a self-explanatory name?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,34 @@
+*.rbc
```
Comment (viktoriia-bilous) [inline .gitignore]: Please remove files and folders, that are not present in your project yet

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, or any plugin's
+// vendor/assets/javascripts directory can be referenced here using a relative path.
+//
+// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
+// compiled file. JavaScript code in this file should be added after the last require_* statement.
+//
+// Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
+// about supported directives.
+//
+//= require rails-ujs
```
Comment (vikabelous) [inline app/assets/javascripts/application.js]: Delete all the comments above

File: app/assets/javascripts/application.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+//
+// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, or any plugin's
+// vendor/assets/javascripts directory can be referenced here using a relative path.
+//
+// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
+// compiled file. JavaScript code in this file should be added after the last require_* statement.
+//
+// Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
+// about supported directives.
+//
```
Comment (viktoriia-bilous) [inline app/assets/javascripts/application.js]: Delete all the comments above

File: app/assets/javascripts/cable.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// Action Cable provides the framework to deal with WebSockets in Rails.
+// You can generate new channels where WebSocket features live using the `rails generate channel` command.
+//
```
Comment (viktoriia-bilous) [inline app/assets/javascripts/cable.js]: Delete all the comments above

File: config/database.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+default: &default
```
Comment (viktoriia-bilous) [inline config/database.yml]: This file should be in .gitignore
You should create a database.yml.example

File: log/development.log [hunk 6]
```diff
@@ -0,0 +1,14 @@
+Started GET "/" for 127.0.0.1 at 2018-03-30 17:18:43 +0300
```
Comment (viktoriia-bilous) [inline log/development.log]: `/log` should be in .gitignore

**Pull request closed without merging**